app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,99 @@
+body {
+  margin: 0;
+}
+
+header {
```

Comment:
everybody is creating own SCSS files

DISCUSS it with all team members and agreed on some structure

Reply:
fixed